bentoml/_internal/bento/docker/templates/_macros.j2

```diff
@@ -38,8 +38,7 @@ set -euxo pipefail
 
 if [ -f {{ path }} ]; then
   echo "{{ echo_msg }}..."
-  chmod +x {{ path }}
-  {{ path }}
+  bash {{ path }}
```

Comment:
does this work for __setup_script__, where user can provide a python script?

Reply:
could you test if this works for this example project? https://github.com/bentoml/gallery/blob/main/custom_runner/nltk_pretrained_model/bentofile.yaml#L8

Reply:
See https://docs.bentoml.org/en/latest/concepts/bento.html#setup-script

"[Shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) .. are required in a setup script provided to BentoML."

Reply:
however, the current implementation for `setup_script` just copies over the user-provided script and does not guarantee the provided script has an execute permission.  we can probably add an `os.chmod`  call to ensure the file permission in Bento directory.  @sauyon what do you think?

Reply:
Maybe for setup scripts we will just use `. setup_script` instead of using bash.

Reply:
Yeah, we should probably do that. Bash scripts / our own python scripts run like this, and user scripts must be executable.

Can the user specify `bash script.sh` as a custom script?

Reply:
I think they can

Reply:
No user can not do that, the setup script must be path to a file